README.md

```diff
@@ -212,11 +212,6 @@ running on another machine which is connected to Alacritty via SSH, this issue
 disappears. Actual throughput and rendering performance are still better in
 Alacritty.
 
-**_My arrow keys don't work._**
-
-It sounds like you deleted some key bindings from your config file. Please
-reference the default config file to restore them.
-
```

Comment:
This section is not true about the recent alacritty, there're no arrow key bindings in a config and you can safely remove them.